src/app/components/ItemFilters/ItemFilters.jsx

```diff
@@ -39,15 +38,26 @@ const ItemFilters = (
 
   // When new items are fetched, update the selected string display.
   useEffect(() => {
+    let timeout;
     setSelectedFieldDisplayStr(parsedFilterSelections());
     // Once the new items are fetched, focus on the filter UI and the
     // results, but don't do this if the user requested to view all items
     // until all the items are fetched. It is annoying if the text keeps
     // getting focused and the page keeps jumping around.
-    if (selectedFieldDisplayStr || finishedLoadingItems) {
+    if (showAll && finishedLoadingItems)  {
```

Comment:
why is `selectedFieldDisplayString` no longer a part of this conditional?

Reply:
The "view all items" feature does not generate the loading screen. The element with the text description is on the page the entire time and will update from "loading more items ..." to "loading more items complete" and it's okay to focus on this right away.

But when a filter or search is performed, the loading screen appears. If we try to focus on the text description element right away, the focus is gone because of the loading screen. So `selectedFieldDisplayStr` is instead in a separate conditional to focus on the element after a second. Yes, the second is somewhat random but it gives enough time on the page for the new items to load and then focus on the element.